feature-libs/user/profile/styles/_new-combine-profile.scss

```diff
@@ -0,0 +1,9 @@
+%cx-new-combined-profile {
+  .new-combined-profile-container {
+    padding-bottom: 5rem;
+    border-width: 1px 0px 1px 0px;
```

Comment:
Oops, does this border really take effect? `px` was used here

Reply:
Yes, you can check the video

Reply:
Looks like this is for the top and bottom line. In this case, there are 2 overlapping lines between 2 sections. And from the video, that overlapping line looks bold than others.

Reply:
Fixed, Please check below snapshot
<img width="1347" alt="image" src="https://github.com/SAP/spartacus/assets/38693320/ffe094ec-a6ef-471b-b744-4dcf738b509b">